config/routes.rb

```diff
@@ -0,0 +1,47 @@
+# Rails.application.routes.draw do
+#   root to: 'users#index'
+#   devise_for :users
+
+#   get '/users/', to: 'users#index'
+#   get '/users/:id/', to: 'users#show'
+
+#   get '/posts/new', to: 'posts#new'
+#   get '/users/:id/posts/', to: 'posts#index'
+#   get '/users/:id/posts/:id/', to: 'posts#show'
+
+#   post '/posts', to: 'posts#create'
+#   post '/users/:user_id/posts/:id/comments', to: 'comments#create'
+#   post '/users/:user_id/posts/:id/likes', to: 'likes#create'
+# end
+
+Rails.application.routes.draw do
+  devise_for :users
+  devise_scope :user do
+    get '/users/sign_out', to: 'devise/sessions#destroy'
+  end
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  root 'users#index'
+  get '/users/', to: 'users#index'
+  get '/users/:id/', to: 'users#show'
+
+  get '/posts/new', to: 'posts#new'
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: 
- [ ] Just as the previous reviewer pointed out, your `devise gem` is still not correctly setup up. When I run `rails db:create`, instead of creating the database, i got an error as shown in the screenshot below. :+1: :100: It seems that devise is not correctly setuo with the user's model, table and routes. I'll suggest you Read through the article the previous reviewer suggested and try to integrate the devise gem correctly :+1: :wave: 

- - Screenshot reference:
![Selection_067](https://github.com/obibaadoma/Blog_App/assets/107632780/b468b87b-6215-42d3-9532-33415cc7c90a)